yt_dlp/extractor/academymel.py

```diff
@@ -0,0 +1,50 @@
+import re
+import time
+
+from .common import InfoExtractor
+from .getcourseru import GetCourseRuIE
+from ..utils import update_url_query, urlencode_postdata
+
+
+class AcademyMelIE(InfoExtractor):
+    _NETRC_MACHINE = 'academymel'
+    _VALID_URL = r'https?://academymel\.online/(?P<id>[^/?#]+)'
+    _LOGIN_URL = 'https://academymel.online/cms/system/login'
+    _TESTS = [{
+        'url': 'http://academymel.online/3video_1',
+        'info_dict': {
+            'id': '4885302',
+            'title': 'Промоуроки Академии МЕЛ',
+            'ext': 'mp4',
+            'duration': 1693
+        }
+    }]
```

Comment:
can you change this test so it tests the playlist, too?

e.g.
```py
    _TESTS = [{
        'url': 'http://academymel.online/3video_1',
        'info_dict': {
            'id': '3video_1',
            'title': 'Промоуроки Академии МЕЛ',
        },
        'playlist_count': 1,
        'playlist': [{
            'info_dict': {
                'id': '4885302',
                # etc
            },
        }]
```

Reply:
Will check if it works. I pulled all your changes, but now the GetCourseRu is failing, but possibly, because I still have a `local_parameters.json` or something. I'm testing.